README.md

```diff
@@ -18,6 +18,12 @@ npm start
 
 Which will start the web application
 
+### Logging in 
+
+To log in you need to provide an IDP or a WebID.
+The application will detect which one you use and log you in.
```

Comment:
It's doesn't log you in. It redirects you to the login page of your identity provider.

Reply:
I have adjusted this in [commit 80e3d2f](https://github.com/SolidLabResearch/generic-data-viewer/pull/52/commits/80e3d2f8509ad3d8f5e7368e35e1205469deaf75)